### DISCUSSION TITLE
broker 再遇到 网络续断的情况是如何处理的

### DISCUSSION THREAD

**Discussion by sdack-cloud** [alibaba/alibaba-rsocket-broker#198] [General]
当请求时出现网络断开  broker 并没有将信息完全传递出去，broker 是如何处理的？
broker 也使用了缓存 如redis ，什么情况下使用缓存

**Comment by linux-china** [comment]
这个需要开发者基于Reactive的失败自行处理。 Broker主要是处理实时通讯，而不是MQ的场景，不会使用本机的内存或者磁盘作为失败请求的缓存。 如果你确认请求失败后要重新发送，Reactive本来就有retry的机制，你可以调用该API或者自己扩展一下就可以处理该场景。
